### ISSUE TITLE
Failed to build aubio, python 3?

### ISSUE THREAD

**Issue by 0xACE** [mxe/mxe#647]
Hello, I got this error.

    Failed to build package aubio!
    ------------------------------------------------------------
        except Errors.WafError ,e:
                               ^
    SyntaxError: invalid syntax

not sure what the proper method of dealing with this is. there seems like there is a fixpy2.py which may fix this problem.

Personally i just temporarily replaced python3 with python2 and it compiled fine.

I noticed that you patch packages so I could post it here...

If you can't get the fixpy2.py to work, this regex can help you replace the exception errors.

    :%s/\v(except [0-9A-Za-z_.]+) ?, ?(\w+?:)/\1 as \2/g

Excuse the poor quality of this report, but I couldn't spend more time on this.

__TL;DR:__ when you've got python3 aubio fails to build with mxe.

**Comment by TimothyGu** [comment]
Waf 1.8 supports Python 3, but aubio has an old internal version of waf. The proper way to fix this is updating the waf toolchain.

**Comment by TimothyGu** [comment]
Python 3 is still not widely supported, and aubio looks dead so closing.
